specs/wp-signup-spec.js

```diff
@@ -814,7 +814,7 @@ testDescribe( `[${host}] Sign Up  (${screenSize}, ${locale})`, function() {
 														this.editor.enterContent( blogPostQuote + '\n' );
 
 														this.postEditorToolbarComponent = new PostEditorToolbarComponent( driver );
-														this.postEditorToolbarComponent.ensureSaved();
+														this.postEditorToolbarComponent.ensureSaved( false );
```

Comment:
If you change the method to use named parameters this will need to be something like: `.ensureSaved( { clickSave: false } );`